src/Backend/SgateShopgatePlugin/Bootstrap.php

```diff
@@ -155,7 +155,7 @@ public function getInfo()
      */
     public function getVersion()
     {
-        return '2.9.88';
+        return '2.9.89';
```

Comment:
```suggestion
        return '2.9.88';
```

-> will be done as part of the release task. 
When PR is done it can be merged, but we will do the release and try to upload after QA approved it.